lib/Tooling/Refactor/Extract.cpp

```diff
@@ -161,6 +161,19 @@ static bool isMultipleCandidateBinOp(BinaryOperatorKind Op) {
   return Op == BO_Add || Op == BO_Sub;
 }
 
+static CompoundStmt::const_body_iterator
+comprehensiveFind(const CompoundStmt *CS, const Stmt *Target) {
```

Comment:
Function names should start with a verb. Maybe this could be findTargetsForExtraction?